packages/__docs__/src/CodePenButton/index.tsx

```diff
@@ -37,31 +38,154 @@ class CodePenButton extends Component<CodePenButtonProps> {
     options: {},
     render: true
   }
-
   render() {
+    // 2 choices:
+    // 1: Add every import manually to every example
+    // 2: Just add render() if needed to every example and do the following:
+    //const code = this.props.code
+    // write something that parses this code, e.g. via regexpes and inserts the imports
+    // for example it looks for "<[Capital letter]" like <Avatar
+    // lets say that it found Avatar and Alert, then it constructs a string like
+    // const result = "import {Avatar, Alert} from '@instructure/ui'"
+    // Search in "code" for the exact string "mirrorHorizontalPlacement", if found
+    // code = "import { mirrorHorizontalPlacement } from '@instructure/ui-position'" + code
+    // add @instructure/ui-position to the package.json dependencies
+    //code = result + code
+    // For step 1 handle the following:
+    // Add React, ReactDOM imports
+    // Add component and icon imports
+    // Add imports for mirrorHorizontalPlacement, DateTime, moment, debounce
+    // And later we'll deal with the image imports and the preview bug
+    const importReact = `
+    import React from "react"
+    import ReactDOM from "react-dom"
+    import moment from 'moment'
+    import {avatarSquare, avatarPortrait, lorem} from "./samplemedia"
+    import {placeholderImage} from "./samplemedia"
+    import iconExample from "!svg-inline-loader!./heart_lg.svg"
```

Comment:
if you remove the spaces from the beginning of these lines the imports will not be indented which looks nicer.